config/locales/new/video_subtitles.de.yml

```diff
@@ -0,0 +1,95 @@
+de:
+  activerecord:
+    attributes:
+      pageflow/text_track_file:
+        kind: Art
+        label: Label
+        srclang: Sprache
+  pageflow:
+    config:
+      text_track_kind:
+        subtitles: Übersetzungs-Textspur
+        captions: Ton-Ersatz-Textspur
+        descriptions: Bild-Ersatz-Textspur
+    editor:
+      errors:
+        nested_type_error: Der Typ '%{type}' des Uploads '%{name}' ist nur für genestete Dateien erlaubt.
+        top_level_type_error: Der Typ '%{type}' des Uploads '%{name}' ist nur für nicht-genestete Dateien erlaubt.
```

Comment:
These error messages are too technical IMO. We cannot expect the user to understand what a nested file is.